src/controllers/showController.test.ts

```diff
@@ -0,0 +1,132 @@
+import * as faker from 'faker';
+
+import { sequelize, Show } from '../models';
+import * as showController from './showController';
+import { ShowInstance } from '../models/Show';
```

Comment:
I don't think you need this line since you don't use `ShowInstance` explicitly in this file!

Reply:
Don't I ShowInstance for createShow (and multiple others)? Or do I not need that function